src/components/hero.jsx

```diff
@@ -49,8 +49,8 @@ const Hero = () => {
 
       <Form />
 
-      <section className="block justify-center  items-center md:flex">
-        <div className="md:w-1/2 md:pr-20 md:text-left text-center">
+      <section className="block justify-center  items-center md:flex md:flex-col-reverse xl:flex xl:flex-row lg:flex lg:flex-row">
+        <div className="lg:w-1/2 px-10 pt-5 lg:px-0 lg:pr-20 md:text-left text-center">
           <p className="text-purple font-bold text-2xl lg:text-3xl dark:text-deeppurple">
```

Comment:
I think adding a `lg:text-center` here would make this text look a bit more uniform in terms of alignment with the one below. What do you think?

Reply:
![image](https://user-images.githubusercontent.com/72051159/216956189-0cec4526-6aaf-44c7-800a-14a53e4707d9.png)

this right?